heat/examples/vm_two_nics_with_share.yaml

```diff
@@ -0,0 +1,72 @@
+heat_template_version: 2018-03-02
+
+description: > 
+    This template illustrates the creation of a VM
+    together with a NFS share
+
+parameters:
+    user_key:
+        type: string
+        label: ssh_user_key
+        description: Public user ssh key to be injected in the cluster VMs
+        constraints:
+            - custom_constraint: nova.keypair
+    user_net1:
+        type: string
+        label: user_net1
+        description: Add the required VM network
+        constraints:
+            - custom_constraint: neutron.network
+    user_net2:
+        type: string
+        label: user_net2
+        description: Add the required VM network
+        constraints:
+            - custom_constraint: neutron.network
+    user_share_name:
+        type: string
+        label: share_name
+        description: Give a name to the share
+        constraints:
+            - length: { min: 1, max: 32 }
+              description: Share name must contain between 1 and 32 characters
+            - allowed_pattern: "[a-zA-Z0-9_]+"
+              description: Only alpha-numeric and "_" characters allowed
+    user_share_size:
+        type: number
+        label: share_size
+        description: Specify the share size (in GB)
+        constraints:
+            - range: { min: 1, max: 100 }
+
+resources:
+    my_server:
+        type: OS::Nova::Server
+        properties:
+          name: My_VM_with_NFS_share
+          metadata:
+              # Some metadata examples to be included in the VM
+              server: master
+              color: red
```

Comment:
maybe you should include an extra metadata value `mount_point: { get_resource: my_share }`

Reply:
Problem: the user don't know the export location path because the share will be created by the same template. That's the problem I wanted to discuss end of last week. I've tried something like this:
`          metadata:
              # Some metadata examples to be included in the VM
              share_export_loc_path: { get_attr: [my_share, export_locations, 0, path] } `
but it gives a 'null' value for share_export_loc_path.

Reply:
ok I understood. OS gives you a  `null`  value if the array or dictionary is not there, probably we are looking in the wrong place.